recipe/meta.yaml

```diff
@@ -10,13 +10,13 @@ source:
 
 build:
     number: 17
```

Comment:
Should we increase the build number to make the distinguishing clear?

Reply:
Didn't want to update `eman-deps`, so kept the build number.